index.js

```diff
@@ -2,17 +2,44 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+//made js render after body loads
 
 /***** Deliverable 2 *****/
-
+header.style.color="#abcdef"
 
 
 /***** Deliverable 3 *****/
 // console.log('This is what the traveler object looks like: ', traveler)
+// travelerNameNode=document.querySelector("h2")
+// travelerNameNode.innerHTML=traveler.name
+
+// travelerNicknameNode=document.querySelector("div em")
+// console.log(travelerNameNode)
+//travelerNameNode.innerHTML=traveler.nickname
+//^ doesn't work, em is inline
+
+profileDiv=document.querySelector("#profile")
+profileDiv.innerHTML=`<img src="${traveler.photo}"
+alt="${traveler.name}">
+<h2>${traveler.name}</h2>
+<em>${traveler.nickname}</em>`
 
 
 /***** Deliverable 4 *****/
+var sightings=traveler.animalSightings
+let sightingList=document.querySelector("#animals")
+for (let i=0;i<sightings.length;i++){
+    let sighting=sightings[i]
+    item=document.createElement("li")
```

Comment:
`const` or `let` 😁  

Random tangent on `const` - There's two schools of thought with `const` in JavaScript. One is to use `const` when you know you are working with values that shouldn't change such as math/physics constants, file versions, etc. Another is to use `const` everywhere and go back and change a declaration to `let` when you come to the point when you need to re-assign that variable.

I tend to follow the second school of thought - I use `const` unless I know I'm going to re-assign a variable later, then I'll use `let`. I do this to eliminate accidental re-assignments.


```suggestion
    const item=document.createElement("li")
```

Reply:
🤔🤔🤔🤔